crates/rolldown_binding/src/options/plugin/binding_builtin_plugin.rs

```diff
@@ -142,6 +146,65 @@ pub struct BindingBuildImportAnalysisPluginConfig {
   pub is_relative_base: bool,
 }
 
+#[napi_derive::napi(object)]
+#[derive(Debug, Deserialize, Default)]
+#[serde(rename_all = "camelCase")]
+pub struct BindingViteResolvePluginConfig {
+  pub resolve_options: BindingViteResolvePluginResolveOptions,
+  pub environment_consumer: String,
+
+  pub runtime: String,
+  pub node_builtins: Vec<String>,
```

Comment:
I am not sure If we need to pass `node_builtins`, there is already one util function exist in the codebase, https://github.com/rolldown/rolldown/blob/dee08ac3d507e0ac2b3f83196295ad5809ede261/crates/rolldown_common/src/ecmascript/node_builtin_modules.rs#L78-L78